bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=dbaas-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.25.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.27.0
```

Comment:
Thanks. Please update your toolchain @recharte

Reply:
I have the same version we have in the main branch, PMM-11681-clusterServiceDNSMode branch has an outdated version.
Should I update the toolchain to the latest v1.28.0?

Reply:
Yep. Go ahead